src/test/resources/log4j.properties

```diff
@@ -1,5 +1,7 @@
-log4j.rootLogger = INFO, out
+log4j.rootLogger = ERROR, out
```

Comment:
I'd suggest WARN instead of ERROR, but +1 on moving to a higher level than INFO.